src/modules/core/components/zoom/zoom.module.scss

```diff
@@ -0,0 +1,41 @@
+.zoom_control {
+  width: 32px;
+  height: 65px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0px 4px 15px 0px rgba(37, 74, 165, 0.1);
```

Comment:
use scss variables from file 'src/styles/variables.module.scss'. In this case use `variables.$main-box-shadow` and check other places where you can use other variables.